rust/stracciatella_c_api/src/c/logger.rs

```diff
@@ -28,6 +28,12 @@ pub extern "C" fn Logger_setLevel(level: LogLevel) {
     Logger::set_level(level)
 }
 
+/// Set log level
```

Comment:
I know this came from me, but there seems to be a copy-paste-error in the comment